src/foldermodel.cpp

```diff
@@ -122,7 +122,7 @@ void FolderModel::loadPendingThumbnails() {
     hasPendingThumbnailHandler_ = false;
     for(auto& item: thumbnailData_) {
         if(!item.pendingThumbnails_.empty()) {
-            auto job = new Fm::ThumbnailJob(item.pendingThumbnails_, item.size_);
+            auto job = new Fm::ThumbnailJob(std::move(item.pendingThumbnails_), item.size_);
```

Comment:
The `item.pendingThumbnails_` can't be referenced/used after this call? I don't have the the snippet of C++ standard to cite, but the [cppreference](http://en.cppreference.com/w/cpp/language/move_constructor) states:

> Move constructors typically "steal" the resources held by the argument (e.g. pointers to dynamically-allocated objects, file descriptors, TCP sockets, I/O streams, running threads, etc), rather than make copies of them, and leave the argument in some valid but otherwise indeterminate state. For example, moving from a std::string or from a std::vector may result in the argument being left empty. However, this behavior should not be relied upon.

So for the sake of correctness, the `item.pendingThumbnails_` should be reinitialized/cleared explicitly.